terraform/iam.tf

```
@@ -121,6 +121,26 @@ data "aws_iam_policy_document" "dynamodb_policy_doc" {
   }
 }
 
+data "aws_iam_policy_document" "transfer_tracker_indexes_access" {
+  statement {
+    actions = [
+      "dynamodb:Query"
+    ]
+    resources = [
+      "arn:aws:dynamodb:${var.region}:${data.aws_caller_identity.current.account_id}:table/${var.environment}-ehr-transfer-tracker/index/*"
```

```suggestion
      "arn:aws:dynamodb:${var.region}:${data.aws_caller_identity.current.account_id}:table/${var.environment}-ehr-transfer-tracker/index/*"
```

`${var.environment}-ehr-transfer-tracker` should come from `data`.